app/components/list_filter_component.rb

```diff
@@ -2,9 +2,10 @@
 
 # View component to add a dialog for filter table by a list
 class ListFilterComponent < Component
-  attr_reader :form
+  attr_reader :form, :samples
```

Comment:
**nitpick (blocking):** Can you rename `samples` to `filters`, as it really isn't a list of samples but a list of filters.

Reply:
Updated in 42bfa4ec7dfbae6991a2175289794d8c0d0d4f03